services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
+|------|----------------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.color.get_report       | null       | Requests a map of color component values.                                                                                                                        |
+| in   | cmd.color.set              | int_map    | Sets values for all color components using a map in which key is a [`color_component`](#definitions) listed in [`sup_components`](#service-properties) property. |
+| out  | evt.color.report           | int_map    | Returns map of color components, where value is the component intensity.                                                                                         |
+| -    |                            |            |                                                                                                                                                                  |
+| in   | cmd.color.start_transition | object     | Starts a single color component transition, see the [`transition`](#definitions) object definition for more details.                                             |
+| in   | cmd.color.stop_transition  | str        | Stop fading/enhancing single color component. val = "red"                                                                                                        |
 
-#### Service props
+## Service properties
 
-Name             | Value example                                                              | Description
------------------|----------------------------------------------------------------------------|-------------
-`sup_components` | ["red", "green", "blue"]                                                   | List of supported color components
-`sup_durations`  | [{"min": 1, "max": 127, "step": 1}, {"min": 180, "max": 7620, "step": 60}] | Supported duration steps for transition change (optional). Min and max always reflects seconds.
+| Name             | Type      | Example                                                                      | Description                                                                                                          |
+|------------------|-----------|------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| `sup_components` | str_array | `["red", "green", "blue"]`                                                   | List of supported color components, see definition of [`color_component`](#definitions) for list of possible values. |
```

Comment:
possible values -> supported values. I would stick to this therm. Possible means that there are more values besides the 'possible' ones ? Supported means that others are not supported.

Reply:
Ok, let's consider it a closed list.

Reply:
The thing is...

A device ***supports*** only a subset of ***possible*** values.

I think possible is a correct word here.

Like ZW device supports only RGB but not temp, cold_w and warm_w.

After thinking this out, I don't think changing is feasible.